doc/BuiltInMetrics.md

```diff
@@ -11,9 +11,11 @@ As of release 1.0.9, metrics are exposed as an experimental feature available at
 | `ExperimentalFeatures__Enabled`          | `true` |
 | `ExperimentalFeatures__EnableMetrics`    | `true` |
 
-### Windows Note
+### Windows Note (1.0.9 only)
 Metrics on Windows are not fully supported the 1.0.9 experimental release. Host metrics (cpu, memory and disk usage) will all show as 0. Moby metrics are supported. In addition, edgeHub must be started as a container administrator to expose metrics. 
 
+This is no longer required in the 1.0.10 release
```

Comment:
Given the title is for 1.0.9 only, do we still need this line?

Reply:
I think it still adds clarity, and it can't hurt

Reply:
I agree. We could either have release specific information in each release branch, or aggregate everything in master. I like the latter because then we don't have to manage multiple branches of docs and it is simpler for customers.